sdk/storage/azure-storage-queue/migration_guide.md

```diff
@@ -0,0 +1,577 @@
+# Storage Queue Service SDK Migration Guide from <= 2.x to 12.x
+
+In this section, we list the main changes you need to be aware of when converting your Storage Queue SDK library from version <= 2.X to version 12.X.
+In version 12 we also support asynchronous APIs.
+
+## Converting Core Classes
+<= 2.X synchronous classes have been replaced. Some functionality has been split into separate clients for more logical organization.
+
+| <= 2.X Classes (Clients)  | V12 Clients | NEW Asynchronous clients |
+|---:|---:|---:|
+| QueueService (account-level operations) | QueueServiceClient | QueueServiceClient |
+| QueueService (queue-level operations) | QueueClient   | QueueClient |
+
+## Version <= 2.X to Version 12 API Mapping
+
+<table border="1" cellspacing="0" cellpadding="0">
+    <tbody>
+        <tr>
+            <td width="353" colspan="2" valign="top">
+                <p align="right">
+                    Version &lt;= 2.X
+                </p>
+            </td>
+            <td width="270" colspan="2" valign="top">
+                <p align="right">
+                    Version 12.X
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="353" colspan="2" valign="top">
+                <p align="right">
+                    Note: Any class means any of QueueService class.
+                </p>
+            </td>
+            <td width="270" colspan="2" valign="top">
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    API Name
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    Class(es) it belongs to
+                </p>
+            </td>
+            <td width="163" valign="top">
+                <p align="right">
+                    API Name
+                </p>
+            </td>
+            <td width="107" valign="top">
+                <p align="right">
+                    Class(es) it belongs to
+                </p>
+            </td>
+        </tr>
+        <tr>
+            <td width="242" valign="top">
+                <p align="right">
+                    generate_account_shared_access_signature
+                </p>
+            </td>
+            <td width="111" valign="top">
+                <p align="right">
+                    Any class
```

Comment:
This is sort of hard to comment on but for the "Class it belongs to", you have "Any Class" for all of Track1 which I don't think is correct. You should put what class the method belongs to here, for example, even from the sample below, it looks like `generate_account_shared_access_signature` is from `QueueService`?